### DISCUSSION TITLE
增加IO功能

### DISCUSSION THREAD

**Discussion by johnnywongxy** [zdz/ServerStatus-Rust#48] [Ideas]
老大你这个项目做的很棒！能否加一个实时硬盘读写的显示啊?
哦对了Alpine-3.16.0-released更新了

**Comment by zdz** [comment]
目前没有计划加这个哦

1. 这个面板不会整天盯着看，就看个大概(一些相对稳定的指标)，磁盘 IOPS 看瞬时值没什么意义，历史记录才有意义，加上了估计又要加读写延迟，读写吞吐，饱和度，IO使用率等等
2. 现在界面显示指标已经差不多了，再加上会显得拥挤，加上了又要考虑到多块磁盘多种类型( SDD HDD) 差异等等情况

有需求的话建议上全家桶  `Prometheus + Grafana + Alertmanager + node_exporter`
